specs/cmd/certificate.md

```diff
@@ -133,6 +134,18 @@ Flags:
   -h, --help       help for generate-test
 ```
 
+### notation certificate cleanup-test
+
+```text
+Clean up a test RSA key and its corresponding certificate that were generated using the "generate-test" command.
+
+Usage:
+  notation certificate cleanup-test [flags] <key_name>
+
+Flags:
```

Comment:
I think you do need user confirmation with the information of the stuffs (e.g. the entries and paths to the files) being cleaned up.

Think about this flag.
```
-y, --yes            do not prompt for confirmation
```